Source/CronosPlayUnreal/Public/DefiWalletCoreActor.h

```diff
@@ -30,6 +30,27 @@ destroyErc1155();
 */
 
 // callback
+// general
```

Comment:
why does it need these delegate declarations for all different types (bool, int, string...)?

Reply:
oh because these general ones are used over it in different functions?

Reply:
maybe good to document it

Reply:
i think  , it's better to rename for each delegate.
(each function have each delegate)
I'll change